### PR TITLE
ACCESS REPORT: import North Carolina 2017-2018 school course offerings

### PR DESCRIPTION
**Pull request by sureshc**
import north carolina school course offerings

@mirlew

### CODE REVIEW

**Comment by sureshc** [comment]
[Import NC Data](https://trello.com/c/nElQH0WB/171-import-nc-data)

File: dashboard/app/models/census/state_cs_offering.rb [hunk 1]
```diff
@@ -36,6 +37,13 @@ def self.construct_state_school_id(state_code, row_hash)
       School.construct_state_school_id('GA', row_hash['SYSTEM_ID'], school_id)
     when 'ID'
       School.construct_state_school_id('ID', row_hash['LeaNumber'], row_hash['SchoolNumber'])
+    when 'NC'
+      # school code in the spreadsheet from North Carolina is prefixed with the district code
```
Comment (breville) [inline dashboard/app/models/census/state_cs_offering.rb]: nit: periods at end of sentences please :)

File: dashboard/app/models/census/state_cs_offering.rb [hunk 2]
```diff
@@ -36,6 +37,13 @@ def self.construct_state_school_id(state_code, row_hash)
...
+      # but our schools data imported from NCES does not
+      district_code = row_hash['NC LEA Code']
+      school_code = row_hash['NC School Code']
+      school_code.slice!(district_code)
```
Comment (breville) [inline dashboard/app/models/census/state_cs_offering.rb]: Maybe this behaviour of `slice!` is well-known, but if you want to save someone like me a few minutes of googling, you could add a comment along the lines of "Remove the district_code from the school_code."

File: dashboard/app/models/census/state_cs_offering.rb (hunk 1 above)
Comment (sureshc) [reply dashboard/app/models/census/state_cs_offering.rb]: Done!

File: dashboard/app/models/census/state_cs_offering.rb (hunk 2 above)
Comment (sureshc) [reply dashboard/app/models/census/state_cs_offering.rb]: Done.

**Pull request merged**